From b153166a512e29b9cc89a420905a8924df74246e Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Wed, 6 Apr 2016 18:32:42 +0300
Subject: [PATCH 1012/1240] Revert "fix: dts: cp110: update CP-100 PINs, to
 support RGMII0"

- Revert this patch after adding support for SGMII in u-boot

This reverts commit e02beeae5b40839afd046d2d7813c10b7be7a84f.

Change-Id: I78fd1166f8592f6f9ce0a1287c43a9e9ef58e0a7
Reviewed-on: http://vgitil04.il.marvell.com:8080/28840
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Neta Zur Hershkovits <neta@marvell.com>
---
 arch/arm/dts/armada-7040-rz-db_router.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/dts/armada-7040-rz-db_router.dts b/arch/arm/dts/armada-7040-rz-db_router.dts
index 2bd33af..bdbd193 100644
--- a/arch/arm/dts/armada-7040-rz-db_router.dts
+++ b/arch/arm/dts/armada-7040-rz-db_router.dts
@@ -91,7 +91,7 @@
 
 			pinctl@440000 {
 					/* MPP Bus:
-					   RGMII0[0-11]
+					   TDM	 [0-12]
 					   SPI   [13-16]
 					   SATA1 [28]
 					   UART0 [29-30]
@@ -104,8 +104,8 @@
 					   SD	 [56-62]
 					*/
 					/*   0   1   2   3   4   5   6   7   8   9 */
-				pin-func = < 3   3   3   3   3   3   3   3   3   3
-					     3   3   0   3   3   3   3   0   0   0
+				pin-func = < 4   4   4   4   4   4   4   4   4   4
+					     4   4   0   3   3   3   3   0   0   0
 					     0   0   0   0   0   0   0   0   9   0xA
 					     0xA 0   7   0   7   0   6   2   2   0
 					     7   7   8   8   1   1   1   1   1   1
-- 
1.9.1

